src/chi/components/tabs/tabs.scss

```
@@ -781,6 +781,19 @@
   }
 }
 
+@mixin chevron-icon {
+  color: inherit;
+  content: '\e91e';
+  display: block;
+  font-family: chi;
```

Let's use the `$font-family-icon` variable here. Using a variable will also resolve an [error in Sonarqube ](https://rules.sonarsource.com/css/RSPEC-4649)which expects at least one generic font family in the value.

```suggestion
  font-family: $font-family-icon;
```